src/Components/Homepage/index.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
+import CarsMain from '../CarsMain'
 
 export const HomePage = () => {
   return (
     <div>
```

Comment:
use main element instead of just a regular div , and every section of the website should be in section element

Reply:
Will change it after I merge the pull requests and add the components into the HomePage.